packages/shared/src/components/auth/OnboardingAnalytics.tsx

```diff
@@ -121,13 +122,20 @@ export const TiktokTracking = (): ReactElement => {
   );
 };
 
-export const trackAnalyticsSignUp = (): void => {
+interface TrackAnalyticsSignUpProps {
+  experienceLevel: keyof typeof UserExperienceLevel;
+}
+
+export const trackAnalyticsSignUp = ({
+  experienceLevel,
+}: TrackAnalyticsSignUpProps): void => {
   if (typeof globalThis.gtag === 'function') {
     globalThis.gtag('event', 'signup');
   }
 
   if (typeof globalThis.fbq === 'function') {
     globalThis.fbq('track', 'signup');
+    globalThis.fbq('track', 'signup_experience_level', experienceLevel);
```

Comment:
Probably only call this if `experienceLevel` is not `undefined`?

Reply:
Just thought we wouldn't have a case like that hopefully since everyone sets experience when they signup... 🤔 but yeah can add that